internal/api/logout.go

```diff
@@ -48,6 +48,7 @@ func (a *API) Logout(w http.ResponseWriter, r *http.Request) error {
 			return terr
 		}
 
+		//exhaustive:ignore Default case is handled below.
```

Comment:
Should we consider moving the default statement into the switch statement and then deleting the comment?

Reply:
I'd leave it as-is, we don't gain that much by it.